src/sass/app_2/_common/base/common.scss

```diff
@@ -1,3 +1,13 @@
+/** @define theme */
+.theme {
+    &--light {
```

Comment:
since this is the most top parent, i guess there is no other way other than put the color manually